wallet/src/main/java/com/android/identity_credential/wallet/PresentationActivity.kt

```diff
@@ -245,122 +321,16 @@ class PresentationActivity : ComponentActivity() {
         state.value = State.NOT_CONNECTED
     }
 
-    private fun error(msg: String) {
+    private fun errorToast(msg: String) {
         Toast.makeText(applicationContext, msg, Toast.LENGTH_SHORT).show()
         this.onDestroy()
     }
```

Comment:
I think it would be more appropriate for this to be called showErrorAndDismiss and also take a `Throwable`... then we can log the throwable using `Logger` (preserving cause, stacktrace etc) and display its `message` using the Toast... IOW, the fact we're showing a toast is just an implementation detail.